README.md

```diff
@@ -69,11 +69,25 @@ with get_sentinel_scene_from_dir(target_dir) as (full_path, ident):
 
 ## Installation
 The easiest way to install `pysat` is through pip. Be aware, that Rasterio requires GDAL >= 1.11, < 3.1.
+
+Most users will want to do this:
+```shell
+pip install ukis-pysat[complete]  # install everything
+```
+
+There's also some lighter versions with less dependencies:
+
 ```shell
-pip install ukis-pysat
+pip install ukis-pysat  # only install core features (ukis_pysat.file)
+
+pip install ukis-pysat[data]  # install dependencies for ukis_pysat.file and ukis_pysat.data
+
+pip install ukis-pysat[raster]  # install dependencies for ukis_pysat.file and ukis_pysat.raster
 ```
 
-in the meanwhile you could do something like this:
+Some helper functions might need additional dependencies like `pandas`, `dask[array]` or `utm`. If this is the case you will receive an `ImportError`.
```

Comment:
Could we provide a "complete" installation option that would install really all dependencies (including pandas, dask, etc.)? So we could have `pip install ukis-pysat` as the standard installation. `pip install ukis-pysat[complete]` to install all dependencies and `pip install ukis-pysat[data]` etc. to install just the dependencies needed for specific modules.

Reply:
We have that, the `dev` installation includes everything, even Sphinx.